editor/editor_asset_installer.cpp

```diff
@@ -132,14 +132,54 @@ void EditorAssetInstaller::open(const String &p_path, int p_depth) {
 
 	Map<String, Ref<Texture>> extension_guess;
 	{
-		extension_guess["png"] = get_icon("ImageTexture", "EditorIcons");
-		extension_guess["jpg"] = get_icon("ImageTexture", "EditorIcons");
-		extension_guess["atlastex"] = get_icon("AtlasTexture", "EditorIcons");
-		extension_guess["scn"] = get_icon("PackedScene", "EditorIcons");
-		extension_guess["tscn"] = get_icon("PackedScene", "EditorIcons");
-		extension_guess["shader"] = get_icon("Shader", "EditorIcons");
-		extension_guess["gd"] = get_icon("GDScript", "EditorIcons");
-		extension_guess["vs"] = get_icon("VisualScript", "EditorIcons");
+		extension_guess["bmp"] = tree->get_icon("ImageTexture", "EditorIcons");
+		extension_guess["dds"] = tree->get_icon("ImageTexture", "EditorIcons");
+		extension_guess["exr"] = tree->get_icon("ImageTexture", "EditorIcons");
+		extension_guess["hdr"] = tree->get_icon("ImageTexture", "EditorIcons");
+		extension_guess["jpg"] = tree->get_icon("ImageTexture", "EditorIcons");
+		extension_guess["jpeg"] = tree->get_icon("ImageTexture", "EditorIcons");
+		extension_guess["png"] = tree->get_icon("ImageTexture", "EditorIcons");
+		extension_guess["svg"] = tree->get_icon("ImageTexture", "EditorIcons");
+		extension_guess["svgz"] = tree->get_icon("ImageTexture", "EditorIcons");
+		extension_guess["tga"] = tree->get_icon("ImageTexture", "EditorIcons");
+		extension_guess["webp"] = tree->get_icon("ImageTexture", "EditorIcons");
+
+		extension_guess["wav"] = tree->get_icon("AudioStreamSample", "EditorIcons");
+		extension_guess["ogg"] = tree->get_icon("AudioStreamOGGVorbis", "EditorIcons");
+		extension_guess["mp3"] = tree->get_icon("AudioStreamMP3", "EditorIcons");
+
+		extension_guess["scn"] = tree->get_icon("PackedScene", "EditorIcons");
+		extension_guess["tscn"] = tree->get_icon("PackedScene", "EditorIcons");
+		extension_guess["escn"] = tree->get_icon("PackedScene", "EditorIcons");
+		extension_guess["dae"] = tree->get_icon("PackedScene", "EditorIcons");
+		extension_guess["gltf"] = tree->get_icon("PackedScene", "EditorIcons");
+		extension_guess["glb"] = tree->get_icon("PackedScene", "EditorIcons");
+
+		extension_guess["gdshader"] = tree->get_icon("Shader", "EditorIcons");
```

Comment:
#49983 hasn't been merged yet, so as-is this cherry-pick is wrong. I fixed this in #49983 though.